hack/run-ci-e2e-test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# TODO: Add steps to execute code added in WINC-65
```

Comment:
Do we want to mention ticket names here? Only because this is public and they are private.

Reply:
I can remove it.